src/components/project/home-block.vue

```diff
@@ -0,0 +1,92 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="project-home-block">
+    <h2 class="title">
```

Comment:
We don't use a ton of `<h*>` elements (and it can be hard to do so in a semantic way correctly). Bootstrap has equivalent classes (for example, `.h2`), but we also don't use those a lot. It might end up being easiest to style these elements separately.